packages/story-editor/src/app/story/storyTriggers/storyEvents/onInitialElementAddedRegister.js

```diff
@@ -46,6 +46,18 @@ function OnInitialElementAddedRegister({ currentStory, dispatchStoryEvent }) {
   return null;
 }
 
-OnInitialElementAddedRegister.propType = registerPropTypes;
+OnInitialElementAddedRegister.propTypes = {
+  currentStory: PropTypes.shape({
+    story: PropTypes.shape({
+      pages: PropTypes.array,
+    }),
+  }),
+  prevStory: PropTypes.shape({
+    story: PropTypes.shape({
+      pages: PropTypes.array,
+    }),
+  }),
+  dispatchStoryEvent: PropTypes.func.isRequired,
+};
```

Comment:
nit: should we maybe check that each page has an elements array since we're referencing that in the `isNewStory(...)` function?